configs/platforms/fedora-f26-x86_64.rb

```diff
@@ -0,0 +1,10 @@
+platform "fedora-f26-x86_64" do |plat|
+  plat.servicedir "/usr/lib/systemd/system"
+  plat.defaultdir "/etc/sysconfig"
+  plat.servicetype "systemd"
+
+  plat.provision_with "/usr/bin/dnf install -y --best --allowerasing autoconf automake rsync gcc make rpmdevtools rpm-libs"
+  plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/yum/pl-build-tools-release-#{plat.get_os_name}-26.noarch.rpm"
```

Comment:
Do we have something similar to `#{plat.get_os_name}` that we could use instead of `26`?

Reply:
I'm fine getting this in as is if we need to, but just wondering

Reply:
there might be, but TBH I don't see much value in that.

We need a different file for each new version anyways, so there's not really much benefit in re-usability there.

Reply:
I find it easier because it means fewer lines to change next time you copy this :) that's why that definitely isn't a blocker, just a thought